src/assets/custom/css/_sass/_announcement-slice.scss

```diff
@@ -1,42 +1,45 @@
 .announcement-slice {
-  background-color: $coral;
+  background-image: url("#{$site-base-url}/assets/custom/img/services/software-modernisation/bg-announcement-slice-softmod.jpg");
   color: white;
-  line-height: 34px;
-  padding-bottom: 10px;
-  padding-top: 10px;
+  padding-bottom: 15px;
+  padding-top: 15px;
   position: absolute;
   width: 100%;
-  z-index: 5;
+  z-index: 9999;
```

Comment:
```suggestion
  z-index: 90
<img width="1440" alt="Screenshot 2020-07-22 at 09 49 47" src="https://user-images.githubusercontent.com/353044/88156233-0e968300-cc01-11ea-9e88-9b589be177a2.png">
;
```

I think you should set this to 90 (or anything below the z-index of the header, which is "100") 

Otherwise you get situations like these
<img width="1440" alt="Screenshot 2020-07-22 at 09 49 47" src="https://user-images.githubusercontent.com/353044/88156297-25d57080-cc01-11ea-88dc-8d1f83b7e778.png">

<img width="1259" alt="Screenshot 2020-07-22 at 09 49 26" src="https://user-images.githubusercontent.com/353044/88156209-05a5b180-cc01-11ea-97ec-444e09d3b32c.png">

Reply:
yeah - that is the proposed solution - it needs to sit lower but the hidden sub nav must not override the CTA - Find out more - which it currently does. overflow hidden sorts this out as per my comment

Reply:
This is now solved : https://github.com/codurance/site/pull/1924/commits/3af7ea09e72e748233756c7769e8c03116608cc4